### PR TITLE
[NG] About: Collaboration Map

### PR DESCRIPTION
**Pull request by leecalcote**
**Design:** https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/?node-id=293%3A0
**Page:** `/company/about`
**Notes:**
The collaboration map disappears in mobile view. Hopefully, someone can fix and push to this PR.

### CODE REVIEW

**Comment by Tanuj22** [comment]
@leecalcote The netlify build needs to be triggered for `layer5-ng` by pushing a commit by the branch was changed. Also for some reason, the prettier in the vs-code seemed to have overridden the eslint pre-commit hook leading to wrong indentations :thinking:

**Comment by leecalcote** [comment]
@Tanuj22 thank you. I have reconfigure eslint in my local environment and had it correct indentation issues that were flagged.

**Comment by Nikhil-Ladha** [comment]
Is the left aligned header intended?
If so, the top margin/padding can be reduced a bit.

File: src/sections/About/about.style.js [hunk 1]
```diff
@@ -20,7 +20,22 @@ const AboutSectionWrapper = styled.section`
       left: -22px;
```
Comment (Nikhil-Ladha) [inline src/sections/About/about.style.js:20]: @leecalcote .....line: 4 in this file `padding: 120px 0 100px 0` is causing that empty white space on the page, if you could change it to `margin: 1rem 0 5rem` that would do the trick...

Comment (leecalcote) [reply src/sections/About/about.style.js:20]: Thank you, @Nikhil-Ladha. I have updated the spacing

**Comment by Nikhil-Ladha** [comment]
Is this PR complete or is there something left to be added?

**Comment by leecalcote** [comment]
Eventually, we'll have another section or two to add to the page, but this PR is good to go for now.

**Comment by Nikhil-Ladha** [comment]
Do you intend to have the globe for mobile view as well?
I can take a look, if you say so.

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by hwpplayer1** [comment]
@Nikhil-Ladha Please do not delete branches I think it is important for archive

**Comment by Nikhil-Ladha** [comment]
And, exactly what kind of archive do you think it's important for?

**Comment by hwpplayer1** [comment]
Maybe I understand wrong or I lose focus. So If you merge it and we can watch git actions with `git log` that is fine. @Nikhil-Ladha

**Comment by Nikhil-Ladha** [comment]
Why do you need the git log of that branch 🤔

**Comment by hwpplayer1** [comment]
No not for me @leecalcote can take care of this branch management @Nikhil-Ladha I wanted to learn how the project is managed I take it serious because of I do research for `release engineering` Of course I do not need any `log`

**Comment by Nikhil-Ladha** [comment]
Cool.....but I am confident enough we don't need an idle/unused branch and anyone is allowed to clean that up. There is no issue with it, and @leecalcote won't mind, me doing that 😅

**Pull request merged**